src/webui/www/private/preferences_content.html

```diff
@@ -1288,68 +1288,68 @@
+        const isQueueingEnabled = $('queueing_checkbox').getProperty('checked');
         $('max_active_dl_value').setProperty('disabled', !isQueueingEnabled);
         $('max_active_up_value').setProperty('disabled', !isQueueingEnabled);
         $('max_active_to_value').setProperty('disabled', !isQueueingEnabled);
         $('dont_count_slow_torrents_checkbox').setProperty('disabled', !isQueueingEnabled);
         updateSlowTorrentsSettings();
     };
 
-    var updateSlowTorrentsSettings = function() {
-        var isDontCountSlowTorrentsEnabled = (!$('dont_count_slow_torrents_checkbox').getProperty('disabled')) && $('dont_count_slow_torrents_checkbox').getProperty('checked');
+    const updateSlowTorrentsSettings = function() {
+        const isDontCountSlowTorrentsEnabled = (!$('dont_count_slow_torrents_checkbox').getProperty('disabled')) && $('dont_count_slow_torrents_checkbox').getProperty('checked');
         $('dl_rate_threshold').setProperty('disabled', !isDontCountSlowTorrentsEnabled);
         $('ul_rate_threshold').setProperty('disabled', !isDontCountSlowTorrentsEnabled);
         $('torrent_inactive_timer').setProperty('disabled', !isDontCountSlowTorrentsEnabled);
     };
 
-    var updateMaxRatioTimeEnabled = function() {
-        var isMaxRatioEnabled = $('max_ratio_checkbox').getProperty('checked');
+    const updateMaxRatioTimeEnabled = function() {
+        const isMaxRatioEnabled = $('max_ratio_checkbox').getProperty('checked');
         $('max_ratio_value').setProperty('disabled', !isMaxRatioEnabled);
 
-        var isMaxSeedingTimeEnabled = $('max_seeding_time_checkbox').getProperty('checked');
+        const isMaxSeedingTimeEnabled = $('max_seeding_time_checkbox').getProperty('checked');
         $('max_seeding_time_value').setProperty('disabled', !isMaxSeedingTimeEnabled);
 
         $('max_ratio_act').setProperty('disabled', !(isMaxRatioEnabled || isMaxSeedingTimeEnabled));
     };
 
-    var updateAddTrackersEnabled = function() {
-        var isAddTrackersEnabled = $('add_trackers_checkbox').getProperty('checked');
+    const updateAddTrackersEnabled = function() {
+        const isAddTrackersEnabled = $('add_trackers_checkbox').getProperty('checked');
         $('add_trackers_textarea').setProperty('disabled', !isAddTrackersEnabled);
     };
 
     // Web UI tab
-    var updateHttpsSettings = function() {
-        var isUseHttpsEnabled = $('use_https_checkbox').getProperty('checked');
+    const updateHttpsSettings = function() {
+        const isUseHttpsEnabled = $('use_https_checkbox').getProperty('checked');
         $('ssl_cert_text').setProperty('disabled', !isUseHttpsEnabled);
         $('ssl_key_text').setProperty('disabled', !isUseHttpsEnabled);
     };
 
-    var updateBypasssAuthSettings = function() {
-        var isBypassAuthSubnetWhitelistEnabled = $('bypass_auth_subnet_whitelist_checkbox').getProperty('checked');
+    const updateBypasssAuthSettings = function() {
```

Comment:
`SyntaxError: redeclaration of const updateBypasssAuthSettings`

These may need to remain on `var` since they remain in the global scope after the Preferences window is closed. Reopening the preferences window triggers the above error.